packages/conversations/__tests__/__dataSets__/put.ts

```diff
@@ -0,0 +1,47 @@
+import { CreateConversationRequest } from '../../lib';
+import {
+  conversation,
+  conversationToCreate,
+  conversationResponse,
+} from '../common';
+
+export default [
+  {
+    label: 'create conversations',
+    requests: [
+      [
+        `/v1/conversations/${conversationResponse.id}`,
+        'PUt',
+        {
+          name: conversationToCreate.name,
```

Comment:
```suggestion
          name: conversationToUpdate.name,
```
I guess it doesn't matter, right?